content/docs/command-reference/stage/list.md

```diff
@@ -10,55 +10,50 @@ usage: dvc stage list [-h] [-q | -v]
                       [targets ...]
 
 positional arguments:
-  targets          Show stages from a dvc.yaml file or a directory (with -R).
-                   'dvc.yaml' by default.
+  targets       Limit command scope to these dvc.yaml files,
+                directories (with -R), or stage names.
+                './dvc.yaml' by default.
 ```
 
-## Description
+> See [`targets`](#options) for more details.
 
-`dvc stage list` is useful for quick listing of stages present in the project
-without having to open `dvc.yaml` files.
+## Description
 
-It provides a list of stages, with a stage name and the description of the
-stage. The description of the stage is read from
-[`desc` field in the `dvc.yaml` file](/doc/user-guide/project-structure/pipelines-files#stage-entries),
-or if it does not exist, DVC generates the short description from the other
-metadata it has about the stage.
+Prints a list of stages including their names and a one-line description (which
+can be omitted using `--names-only`). This command is useful for discovering or
+reviewing what stages are present in the project without having to examine
+`dvc.yaml` files manually. It can also be a shorter alternative to `dvc dag`.
```

Comment:
shorter sounds like command itself is shorter, also, I doubt this reference is important, let's focus on the list itself?

Reply:
Removed `dag` ref.